ruby-challenges.rb

```diff
@@ -10,16 +10,65 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def removeItem (arr, str)
+  arr.filter {|elements| 
+          elements.include?(str)}
```

Comment:
Logic looks great here.  When using curlies in Ruby, it is best practice to keep it on the same line, otherwise, you can use a do and end.